packages/core/src/components/ActivityIndicator.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import {
+  StyleProp,
+  ViewStyle,
+  ActivityIndicator as ActivityIndicatorRN,
+  View,
+  Platform,
+} from "react-native";
+import { withTheme } from "@draftbit/theme";
+import type { ReadTheme } from "@draftbit/theme";
+import {
+  Bounce,
+  Chase,
+  Circle,
+  CircleFade,
+  Flow,
+  Fold,
+  Grid,
+  Plane,
+  Pulse,
+  Swing,
+  Wander,
+  Wave,
+} from "react-native-animated-spinkit";
+
+export enum ActivityIndicatorType {
+  plane = "plane",
```

Comment:
```suggestion
export enum ActivityIndicatorType {
  default = "default"
  plane = "plane",
```
Let's have a default type to make it more explicit